shardy/dialect/sdy/ir/attrs.td

```diff
@@ -29,6 +29,11 @@ include "shardy/dialect/sdy/ir/dialect.td"
 // split.
 def Sdy_ManualAxes : ArrayOfAttr<Sdy_Dialect, "ManualAxes",
                                  "manual_axes", "StringAttr"> {
+  let mnemonic = "manual_axes";
```

Comment:
I'm not sure about this change, in `ArrayOfAttr` we already specify the mnemonic, and eltName which creates a parameter. If we want to go with this change we shouldn't use ArrayOfAttr anymore, just a regular AttrDef.

@bartchr808 wdyt?

Reply:
Oh true! Hmm I think going for AttrDef may be better, to let use specify the summary.

Reply:
I think I added this just for consistency as I was checking things, happy to revert if this is not something we want to do right now.

Reply:
Oh wait, can you remove mnemonic and parameters, but keep summary, will that work?